setup/src/main/java/io/jenkins/jenkinsfile/runner/JenkinsEmbedder.java

```diff
@@ -316,6 +316,8 @@ protected Hudson newHudson() throws Exception {
         jettyLevel(Level.WARNING);
         ServletContext webServer = createWebServer();
         File home = homeLoader.allocate();
+        setupHome(home);
+        // TODO looks like a remaining of Jenkins Test Harness not used at all in the jfr context
```

Comment:
Yes. I had a plan to use it for additional extension points inside Jenkinsfile Runner, but I have not got to it yet